book/routing.rst

```diff
@@ -564,7 +565,7 @@ Take a quick look at the routes that have been created so far:
         class BlogController extends Controller
         {
             /**
-             * @Route("/blog/{page}", defaults={"page" = 1})
+             * @Route("/blog/{page}", name="blog", defaults={"page" = 1})
```

Comment:
Hmm, I see you reference it below - so we can keep it - by how about `blog_list` to be a little less generic sounding?